packages/react-core/src/components/Skeleton/examples/Skeleton.md

```diff
@@ -0,0 +1,121 @@
+---
+id: Skeleton
+section: components
+cssPrefix: pf-c-skeleton
+propComponents: ['Skeleton']
+beta: true
+---
+
+## Examples
+
+### Default
+
+```js
+import React from 'react';
+import { Skeleton } from '@patternfly/react-core';
+
+<Skeleton />;
+```
+
+### Percentage widths
+
+```js
+import React from 'react';
+import { Skeleton } from '@patternfly/react-core';
+
+<React.Fragment>
+  <Skeleton width="25%" />
+  <br />
+  <Skeleton width="33%" />
+  <br />
+  <Skeleton width="50%" />
+  <br />
+  <Skeleton width="66%" />
+  <br />
+  <Skeleton width="75%" />
+  <br />
+  <Skeleton />
+</React.Fragment>;
+```
+
+### Percentage heights
+
+```js
+import React from 'react';
+import { Skeleton } from '@patternfly/react-core';
+
+<div style={{ height: '400px' }}>
+  <Skeleton height="25%" style={{ width: '15%', display: 'inline-block', marginRight: '4px' }} />
```

Comment:
why not use the `width` prop? Also i'd remove the inline styles from the Skeleton components and move them to the parent just so it's a bit cleaner, something like
```
<div style={{ height: '400px', display: 'flex', alignItems: 'flex-end', justifyContent: 'space-between' }}>
  <Skeleton height="25%" width="15%" />
  <Skeleton height="33%" width="15%" />
  <Skeleton height="50%" width="15%" />
  <Skeleton height="66%" width="15%" />
  <Skeleton height="75%" width="15%" />
  <Skeleton height="100%" width="15%" />
</div>;
```